erc20/erc20-deploy.js

```diff
@@ -55,47 +56,69 @@ async function deploy(info) {
+  var firstChildOperator = "";
+  for (const url of conf.url.children) {
+    log = 'querying parentOperator to the child node'
+    const parentOperatorRes = await jsonRpcReq(url, log, 'subbridge_getParentOperatorAddr', []);
+    log = 'querying childOperator to the child node'
+    const childOperatorRes = await jsonRpcReq(url, log, 'subbridge_getChildOperatorAddr', []);
+    const parentOperator = parentOperatorRes.data.result;
+    const childOperator = childOperatorRes.data.result;
+    if (firstParentOperator == "") {
+      firstParentOperator = parentOperator;
+      firstChildOperator = childOperator;
+    }
+
+    // register operator
+    await conf.contract.child.newInstanceBridge.methods.registerOperator(childOperator).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+    await conf.contract.parent.newInstanceBridge.methods.registerOperator(parentOperator).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
 
-  // transferOwnership
-  await conf.child.newInstanceBridge.methods.transferOwnership(conf.child.operator).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.transferOwnership(conf.parent.operator).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+    log = 'registering bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_registerBridge', [conf.contract.child.bridge, conf.contract.parent.bridge]);
+
+    log = 'subscribing bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_subscribeBridge', [conf.contract.child.bridge, conf.contract.parent.bridge]);
+
+    log = 'register token to subbridge..'
+    await jsonRpcReq(url, log, 'subbridge_registerToken', [conf.contract.child.bridge, conf.contract.parent.bridge, conf.contract.child.token, conf.contract.parent.token]);
+    /* Above 3 APIs can also be called via Klaytn Javascript Console.
+     * > subbridge.registerBridge("0xCHILD_BRIDGE_ADDRESS", "0xPARENT_BRIDGE_ADDRESS")
+     * > subbridge.subscribeBridge("0xCHILD_BRIDGE_ADDRESS", "0xPARENT_BRIDGE_ADDRESS")
+     * > subbridge.registerToken("0xCHILD_BRIDGE_ADDRESS", "0xPARENT_BRIDGE_ADDRESS", "0xCHILD_TOKEN_ADDRESS", "0xPARENT_TOKEN_ADDRESS")
+     */
+  }
+
+  // set operator threshold
+  // In this example, we set operator threshold to the number of bridge operators,
+  // so all bridge must be operational in order for value transfer to succeed.
+  // This is intended to check if all bridges are working fine.
+  // In production, use an appropriate threshold to fit your needs.
+  await conf.contract.child.newInstanceBridge.methods.setOperatorThreshold(0, conf.url.children.length).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstanceBridge.methods.setOperatorThreshold(0, conf.url.children.length).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+  // transfer ownership to first operator
```

Comment:
Is there a special reason why the first operator takes the bridge contract's ownership?

Reply:
It is easier to find out which operator took the ownership.

If the user wants to make changes to the bridge contract (e.g., change threshold), the owner account key has to be acquired from the child node (because the ownership was transfered). If we choose a random operator from the list, the user should take additional step to find out which operator took the ownership.

Reply:
Okay. Thanks.